app/services/terms_of_use/acceptor.rb

```diff
@@ -2,8 +2,9 @@
 
 module TermsOfUse
   class Acceptor
-    def initialize(user_account:, version:)
+    def initialize(user_account:, common_name:, version:)
```

Comment:
It looks like this file doesn't have a CODEOWNERS entry

Reply:
Would you mind rebasing from master. I made a change to [cover both cases for CODEOWNERS directory definitions here](https://github.com/department-of-veterans-affairs/vets-api/pull/13803)